_posts/2020-09-01-on-air-next-our-podcast.md

```diff
@@ -0,0 +1,30 @@
+---
+title:  "On air next: our podcast!"
+date: 2020-09-01
+author: Alba Roza
+type: blogpost
+excerpt: Launch of the livestream/videopodcast project
+category:
+  - Codebase stewardship
+  - Communications
+---
+
+# On air next: our podcast!
+
+A community couldn't exist without knowledge sharing. Indeed, this is one of its most powerful and appealing characteristics, although it's obvious that not all of its members can invest the same number of hours collaborating in a project. 
+
+Sharing assets is something we always keep in mind at the Foundation for Public Code. We've always tried to share as many guides, tools and tips as we can in order to nurture our community. However there was something we were lacking, an even more direct way of communicating with our community: a live, closer and bilateral communication.
+
+This is why we are **launching our first live stream/podcast, 'Let's talk about public code'**, a space in which we'll interview people strongly related to popular public and open codebases, 'public code' ones.
+
+![Listen to our podcast in our Youtube channel]({{site.url}}/assets/podcast-available-youtube-channel.jpg)
```

Comment:
Shouldn't this link to one/all of the livestream platforms, so people can pick their fav? That way it makes more sense to say later on that you can catch it on YouTube if you didn't make it.
(I assume most people consider YouTube to be for pre-recorded content, so we should make extra clear where/how people can view the live version.)

Reply:
You're right. Let's add it in the end of the blog post.